src/main/main.c

```diff
@@ -3,20 +3,23 @@
 #include <libgte.h>
 #include <libgpu.h>
 #include "linker.h"
-#include "libgv/libgv.h"
+#include "psyq.h"
```

Comment:
To ensure that all necessary headers are included and to avoid any missing dependencies that might be indirectly included by other files, it might be beneficial to sort the includes in the following order:

1. The corresponding header file of the `.cpp`.
2. Project-specific headers.
3. Headers for third-party dependencies.
4. Standard library headers.

This approach can help to catch any missing includes more easily.

Reply:
I can get behind at least including a module's corresponding public header first; that's what I intend to for the rest of the includes.

I've mostly rearranged the includes in main.c to reflect the tiered structure of MGS's engine.
Names taken from The Document of MGS2 (in order from lowest level):
- Hardware: low-level SDK libraries
- Kernel: Multi-Task Scheduler, sound driver
- System: Project-specific service libraries (libgv, libfs, libdg, etc.)
- Game: Game-state/flow management (game)
- User: Various actors.